src/components/Header/HeaderTabs.jsx

```diff
@@ -0,0 +1,57 @@
+"use client";
+import { useState } from "react";
+import { Tab } from "@mui/material";
+import { Tabs } from "@mui/material";
+import ChevronRightIcon from "@mui/icons-material/ChevronRight";
+
+const HeaderTabs = ({ tabList, tabColor, tabActiveColor, iconPlace }) => {
+  const [value, setValue] = useState("");
+
+  const handleChange = (event, newValue) => {
+    setValue(newValue);
+  };
+  return (
+    <Tabs
+      TabIndicatorProps={{
+        display: "none",
+      }}
+      value={value}
+      onChange={handleChange}
+      style={{ height: "38px", display: "flex", alignItems: "center" }}
+    >
+      {tabList.map((item) => (
+        <Tab
+          value={item.value}
+          key={item.name}
+          label={item.name}
+          icon={<ChevronRightIcon sx={{ transform: "rotate(90deg)" }} />}
+          iconPosition="end"
+          disableRipple
+          sx={{
```

Comment:
It's better to move all styling to scss files

Reply:
Previously when we tried to use classNames on MUI elements it didn't work, so MUI styling stays with MUI element.